Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -127,20 +127,20 @@ void RenderFlexibleBox::computeIntrinsicLogicalWidths(LayoutUnit& minLogicalWidt
     // https://bugs.webkit.org/show_bug.cgi?id=116117 and
     // https://crbug.com/240765.
     size_t numItemsWithNormalLayout = 0;
-    for (RenderBox* flexItem = firstChildBox(); flexItem; flexItem = flexItem->nextSiblingBox()) {
-        if (flexItem->isOutOfFlowPositioned() || flexItem->isExcludedFromNormalLayout())
+    for (RenderBox& flexItem : childrenOfType<RenderBox>(*this)) {
```

Comment:
use auto please.

Reply:
I think Alan might means - `for (auto& flexItem : childrenOfType<RenderBox>(*this))` rather than just `auto flexItem`.

Reply:
> @alanbaradlay This function signature has const at the end, which is why it's failing to compile when using `childrenOfType`. Should I remove the const, or is it better to keep it?

oic, `this` function is `const` which means  `childrenOfType<RenderBox>(*this)` returns `const RenderBox&` and we fail at e.g. `flexItem.layoutIfNeeded()` as it expects a `non-const` flexItem. You can't change the signature of this function.